content/es/docs/reference/glossary/sysctl.md

```diff
@@ -0,0 +1,24 @@
+---
+title: sysctl
+id: sysctl
+date: 2019-02-12
+full_link: /docs/tasks/administer-cluster/sysctl-cluster/
+short_description: >
+  Una interfaz para obtener y asignar valores a parámetros del núcleo Unix activo.
+
+aka:
+tags:
+- tool
+---
+ `sysctl` es una interfaz común usadapara consultar o modificar artributos del 
+ núcleo Unix durante su ejecución.
+
+<!--more-->
+
+En los sistemas Unix-like, `sysctl` es el comando que usan los administradores,
```

Comment:
“Unix-like”?

Reply:
I was socked when the Spanish did't translate the term so I think if it is good enough for the Wikipedia https://es.wikipedia.org/wiki/Unix-like it  shall be for us